sbi/utils/mcmc_utils.py

```diff
@@ -0,0 +1,82 @@
+# This file is part of sbi, a toolkit for simulation-based inference. sbi is licensed
+# under the Apache License Version 2.0, see <https://www.apache.org/licenses/>
+
+import logging
+from typing import Callable, Optional, Union
+
+from numpy import ndarray
+from torch import Tensor
+
+from sbi.samplers.mcmc import SliceSamplerSerial, SliceSamplerVectorized
+from sbi.utils.torchutils import tensor2numpy
+
+
+def get_mcmc_samples(
+    potential_fn: Callable,
+    initial_params: Union[ndarray, Tensor],
+    num_samples: int = 1000,
+    vectorized: bool = True,
```

Comment:
That's great! I think 99% of the time we do use one of the slice samplers, but in principle can we expose any of our mcmc samplers here with a `sampler` arg?